x-pack/plugins/observability/public/hooks/use_alert_bulk_case_actions.ts

```diff
@@ -38,7 +37,7 @@ export const useBulkAddToCaseActions = ({ onClose, onSuccess }: UseAddToCaseActi
   });
 
   return useMemo(() => {
-    return hasWritePermissions
+    return casePermissions.create && casePermissions.read
```

Comment:
No not at the moment. We'll likely need to refactor this in the future if we split out `create`. This should probably be:

`add to new case` -> must have create (we might be able to argue that you don't need read)
`add to existing case` -> must have create (because we're creating a comment) and read (otherwise you can't select which case to attach the alerts to)